ipaclient/install/client.py

```diff
@@ -3529,9 +3538,21 @@ def __init__(self, **kwargs):
             raise RuntimeError(
                 "--server cannot be used without providing --domain")
 
-        if self.force_ntpd and self.no_ntp:
+        if self.force_ntpd:
+            raise RuntimeError(
+                "--force-ntpd has been obsoleted by --force-chrony")
+
+        if self.force_chrony and self.no_ntp:
+            raise RuntimeError(
+                "--force-chrony cannot be used together with --no-ntp")
+
+        if self.ntp_servers and self.no_ntp:
+            raise RuntimeError(
+                "--ntp-server cannot be used together with --no-ntp")
+
+        if self.ntp_pool and self.no_ntp:
             raise RuntimeError(
-                "--force-ntpd cannot be used together with --no-ntp")
+                "--ntp-pool cannot be used together with --no-ntp")
```

Comment:
What's the difference between `--ntp-servers` and `--ntp-pool`?

Reply:
The --ntp-server (knob ntp_servers) is used to specify a exactly one time server and may be used multiple times. --ntp-pool option is meant to be pool of multiple servers resolved as one hostname. This pool's servers may vary but pool address will be still same and chrony will choose only one server from this pool.

Reply:
Is it safe to assume that a user will be familiar enough with chrony to know the difference? Should it be pointed out in the man page at least?

Reply:
It will be better to have it in man page, sure.

Reply:
Where exactly should I add it please?

Reply:
To all programs that inherit this. From what I can tell this is at least ipa-client-install, ipa-server-install and ipa-replica-install.